components/intake.py

```diff
@@ -1,14 +1,16 @@
 from rev import CANSparkMax
-from wpilib import DoubleSolenoid, PneumaticsModuleType
+from wpilib import DoubleSolenoid, DigitalInput, PneumaticsModuleType
 from magicbot import tunable
 import ids
 
 
 class Intake:
+    ids = ids
```

Comment:
Wait, what's this doing here?

Reply:
There were multiple classes in ids.py so I thought I could access them and their member variables by doing this.
How should have I done it?

Reply:
lmao

Reply:
Addressed in commit. (hahah)